.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+-   id: pycodestyle
+    name: pycodestyle
+    description: 'pycodestyle (formerly called pep8) - Python style guide checker'
+    entry: pycodestyle
+    language: python
+    types: [python]
+    require_serial: true
```

Comment:
I don't believe `require_serial: true` makes any sense for pycodestyle.

Reply:
You are right. I removed it